### PR TITLE
Build multi-platform bins for Linux, Mac Windows

### LINKED ISSUE #13: Build multi-platform binaries for Linux, Mac and Windows
#### Description

Build binary/executables for Linux, Mac, Windows, *BSD for 32-bit, 64-bit, ARM-6 (that covers Raspberry Pi A, A+, B, B+, Zero), ARM-7(that is RPi2 and RPi3), ARM64 (that is RPi4).

#### Acceptance Criteria

- [x] Use goreleaser to build all of those on the CI (GH Actions Workflow)
- [x] Make sure they get published when there's a new release

Complexity: 2

### LINKED ISSUE #15: Reproducible builds
#### Description

To make sure releases, checksums and signatures are reproducible, make some of the following modifications listed in https://goreleaser.com/customization/build/#reproducible-builds

#### Acceptance criteria

- [x] Read and implement [gorelease reproducible buils](https://goreleaser.com/customization/build/#reproducible-builds) configuration

Complexity: 1

### PR DESCRIPTION
**Pull request by cerebrux**
1. Added missing "popular" BSD OSes (dragonfly and netbsd)
2. Added arm6 and arm7 binaries
3. Added modified timestamp on the output binary to ensure a build was reproducible (see: https://goreleaser.com/customization/build/)
4. Added ignore section, for architecture that are not supported by Go in
combination with the $goos (see https://golang.org/doc/install/source#environment)

Fixes #13
Fixes #15

### CODE REVIEW

**Comment by cerebrux** [comment]
And as usual my test **https://github.com/cerebrux/romie/releases/tag/0.0.1**

**Comment by drpaneas** [comment]
@GrSoul  was working on https://github.com/drpaneas/romie/issues/15
@cerebrux is this covered already by your PR?

File: .goreleaser.yml [hunk 1]
```diff
@@ -8,10 +8,27 @@ builds:
...
+      - goos: freebsd
+        goarch: arm64
+      - goos: netbsd
+        goarch: arm64
+      - goos: dragonfly
+        goarch: 386, arm, arm64
+    goarm:
+      - 6
+      - 7
+    mod_timestamp: "{{ .CommitTimestamp }}"
```
Comment (drpaneas) [inline .goreleaser.yml:32]: This line `fixes #15` as well. Please add that into your in your commit msg.

e.g. `git commit --amend`.

**Review by drpaneas — CHANGES_REQUESTED** [review]
Add the information that fixes `15 issue` as well.

File: .goreleaser.yml (hunk 1 above)
Comment (cerebrux) [reply .goreleaser.yml:32]: Done. Number 3 in my comment describes the commit. Also added that it fixes #15

**Review by drpaneas — APPROVED** [review]

**Review by zoispag — APPROVED** [review]
Changes make sense. Good call on the `ignore` section to only produce valid combinations.

**Pull request merged**